packages/client/src/forms/mappings/mutation/field-mappings.ts

```diff
@@ -638,3 +638,29 @@ export const longDateTransformer =
 
     return transformedData
   }
+
+export const childOsiaUinToIdentityTransformer = (
+  transformedData: TransformedData,
+  draftData: IFormData,
+  sectionId: string
+) => {
+  const sectionData = transformedData[sectionId]
```

Comment:
@rabiulislamanik How about we do this?
```ts
if (!draftData[sectionId].childOsiaUin) {
  return
}
```
That way there won't be an extra empty identifier in the child payload